students/JerryH/Lesson08/circle.py

```diff
@@ -0,0 +1,57 @@
+import math
+
+class Circle:
+
+    def __init__(self, radius):
+        self.radius = radius
+        self._area = radius ** 2 * math.pi
+
+    @property
+    def diameter(self):
+        return self.radius * 2
+
+    @diameter.setter
+    def diameter(self, value):
+        self.radius = value / 2
+
+    @property
+    def area(self):
+        """ Read Only """
+        return self._area
```

Comment:
note that to make these attributes really read only you need to use double underscore -> `self.__area`

something like this will change the value of area:
```
c._area = 4
```